index.tsx

```diff
@@ -0,0 +1,353 @@
+import React, {PropsWithChildren, useCallback, useMemo, useRef} from 'react';
+import {LayoutChangeEvent, StyleProp, StyleSheet, View, ViewProps} from 'react-native';
```

Comment:
Please add rule to remove ";"

Reply:
Are you sure about that? usually, TS developers want to have that due to more readability.